src/services/chat.ts

```diff
@@ -0,0 +1,72 @@
+import { ref } from 'vue';
+import { ChatParticipant, ChatElement, ChatColor } from '@/models/chat';
+
+export const chatHistory = ref(Array<ChatElement>());
+const messageDelay = 1000;
+
+export function sendStartMessgae() {
+  chatHistory.value.push({ from: ChatParticipant.OTHER, message: 'Hey', color: ChatColor.LIGHT });
+  chatHistory.value.push({
+    from: ChatParticipant.OTHER,
+    message: "I'm having big trouble with this code. Can you help me finding the bug?",
+    color: ChatColor.LIGHT,
+  });
+}
+
+export async function sendSubmitMessage(success: boolean, gameFinished: boolean) {
+  sendMessage({
+    from: ChatParticipant.ME,
+    message: 'I think I found the bug. Is the programm now running?',
+    color: ChatColor.INFO,
+  });
+  if (success) {
+    await sendDelayedMessage({
+      from: ChatParticipant.OTHER,
+      message: 'Yes it works! Thank you very much',
+      color: ChatColor.SUCCESS,
+    });
+  } else {
+    await sendDelayedMessage({
+      from: ChatParticipant.OTHER,
+      message: 'No sadly not.',
+      color: ChatColor.WARNING,
+    });
+  }
+  if (!gameFinished) {
+    await sendDelayedMessage({
+      from: ChatParticipant.OTHER,
+      message: 'Can you help me again with another bug?',
+      color: ChatColor.LIGHT,
+    });
+  } else {
+    await sendDelayedMessage({
+      from: ChatParticipant.OTHER,
+      message: 'Thank you a lot. You helped me fixing all my codes!',
```

Comment:
```suggestion
      message: 'Thank you a lot. You helped me a lot!',
```

Reply:
Is 

> Thank you **a lot**. You helped me **a lot**

really a good idea?

Maybe we simply remove the `all` to be valid if the player has one correct answer?

Reply:
I'm fine with both options as long it isn't the current approach.

Reply:
Ok I remove the `all`